student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    
+    // Added comparison between hard coded username and the username obtained from malloc_username_from_conf_file
+    TEST_ASSERT_TRUE_MESSAGE(!(my_username() == malloc_username_from_conf_file()),"AESD students, please fix me!");
```

Comment:
@aasu8675 will this logic check work to compare two strings?  What will the `==` compare here?

Also what is needed for the string returned by `malloc_username_from_conf_file`?